ec_api/apps/api_proxy/views.py

```diff
@@ -0,0 +1,32 @@
+from django.conf import settings
+from django.utils.module_loading import import_string
+from rest_framework.views import APIView
+from rest_framework.response import Response
+
+from api_proxy.upstream_api_client import ResponseBuilderApiClient
+
+
+def get_upstream_client():
```

Comment:
I think I follow what happens here but a docstring would be useful to explicitly state that the `ResponseBuilderApiClient` is only used in tests?